experiments/68x50/slotnick_etal_2001/uber_slotnick_all.m

```diff
@@ -4,11 +4,11 @@
 stats={}; %distns
 plots=stats;
 
-[args,opts] = uber_kitterle_args( 'plots', plots, 'stats', stats);
+[args,opts] = uber_slotnick_args( 'plots', plots, 'stats', stats, 'runs', 5, 'ac.AvgError', 2E-4);
```

Comment:
Please change `runs` to something more reasonable. That value is good for development, but not good for final results.